gruntfile.js

```diff
@@ -75,37 +68,6 @@ module.exports = function( grunt ) {
 			},
 		},
 
-		uglify: {
-			options: {
-				ASCIIOnly: true,
-			},
-			all: {
-				expand: true,
-				cwd: 'src/js/',
-				src: '**/*.js',
-				dest: 'dist/js/',
-				ext: '.min.js',
-			},
-		},
```

Comment:
This function is actively used to minify source scripts and place them into the `dist/js` folder on the build. 
For example `src/js/coblocks-lightbox.js` or `src/js/coblocks-masonry.js` are minified and placed into `dist/js`.
This will be handled or unnecessary with the new build process I presume?

Reply:
@AnthonyLedesma This should be handled by `npm` or `wp-scripts`